assets/style.scss.liquid

```diff
@@ -1837,7 +1837,7 @@ button.drop-shadow {
 /*================ Text wrapper ================*/
 
 .text-max-width {
```

Comment:
This change was requested by marketing and the find a store page necessitated it. I've done a quick spot check that no pages are broken but be aware of it.